index.html

```diff
@@ -135,11 +135,12 @@ <h1>Automatic License Plate Recognition</h1>
     <div class="row">
     <div class="col-md-6">
     <h3> <span class="glyphicons glyphicon glyphicon-map-marker"></span> Where</h3> 
-    <p>Unknown since the ALPRs are mobile. Documents acquired by the author in 2015 show that there were at least 35 vehicles in the Chicago Police Department that had ALPRs mounted to them. Chicago can also use their video surveillance network to scan license plates.</p>
+    <p>Unknown since the ALPRs are mobile. In 2019 the Chicago Police Department purchased an additional <a href ="https://news.wttw.com/2019/01/08/chicago-police-adding-200-plate-reader-vehicles-help-combat-carjackings"
```

Comment:
We should add a sentence that LPL has lawsuit for the locations of fixed-location ALPRs. The case number is 2020 CH 01928 however do not link to our complaint.